neuralprophet/df_utils.py

```diff
@@ -90,6 +90,34 @@ def join_dataframes(df_dict):
     return df_joined, episodes
 
 
+def check_n_lags_and_n_covars(config_covar, n_lags):
```

Comment:
you could make it clearer what the function does with the name, e.g. `get_max_num_lags`

Reply:
you could also simplify the function to sth like:
return max([n_lags] + [val.n_covars for key, val in config_covar.items()]

Reply:
Done